Makefile

```diff
@@ -19,8 +19,8 @@ ifndef VERSION
 VERSION := $(shell echo `git rev-parse --abbrev-ref HEAD`-`git log -1 --pretty=format:%h`-`date "+%d.%b.%Y.%H.%M.%S"`)
 endif
 
-REGISTRY ?= dpcpinternal
-JUMP_PAD_IMAGE = $(REGISTRY)/jump-pad
+REGISTRY ?= gcr.io/velero-gcp
```

Comment:
Why it's gcp spesific?

Reply:
After discussion in meeting, change to GCR is a simpler solution by now.